src/driver/odoc.ml

```diff
@@ -55,11 +55,6 @@ let compile ~output_dir ~input_file:file ~includes ~suppress_warnings ~parent_id
   in
   let cmd = cmd % "--parent-id" % Id.to_string parent_id in
   let cmd = if suppress_warnings then cmd % "--suppress-warnings" else cmd in
-  let dirname = Id.to_fpath parent_id |> Fpath.filename in
-  let cmd =
-    if Fpath.filename file = "index.mld" then cmd % "--short-title" % dirname
```

Comment:
The `--short-title` option is not used anymore but still defined. Shouldn't we completely remove it?